corehq/apps/userreports/models.py

```diff
@@ -541,6 +541,21 @@ def validate_db_config(self):
     def data_domains(self):
         return [self.domain]
 
+    @property
+    def is_rebuild_in_progress(self):
+        return (
+            not self.is_static
+            and (
+                self.meta.build.initiated
+                or self.meta.build.initiated_in_place
+            )
+            and (
+                not self.meta.build.finished
+                and not self.meta.build.finished_in_place
+                and not self.meta.build.rebuilt_asynchronously
+            )
```

Comment:
I think these should move to `DataSourceBuildInformation` or the `DataSourceMeta` classes.

Reply:
It also looks like this info is already available in the view and used elsewhere in the template: https://github.com/dimagi/commcare-hq/blob/b157ed6803640c5da69580b1984991a581f45571/corehq/apps/userreports/views.py#L1138-L1148